cardano-api/src/Cardano/Api/Query.hs

```diff
@@ -61,18 +65,21 @@ import qualified Ouroboros.Consensus.HardFork.Combinator.AcrossEras as Consensus
 import qualified Ouroboros.Consensus.HardFork.Combinator.Degenerate as Consensus
 import qualified Ouroboros.Consensus.HardFork.History as History
 import qualified Ouroboros.Consensus.HardFork.History.Qry as Qry
+import qualified Ouroboros.Consensus.Ledger.Query as Ledger
```

Comment:
This is qualified twice. Is there a reason why?

Reply:
Removed!